src/types.rs

```diff
@@ -1555,10 +1610,16 @@ impl ReadSessionRequest {
         Self { limit, ..self }
     }
 
+    /// Overwrite heartbeats.
+    pub fn with_heartbeats(self, heartbeats: bool) -> Self {
+        Self { heartbeats, ..self }
+    }
+
```

Comment:
Let's leave this out for now, till we can also handle the heartbeat messages https://github.com/s2-streamstore/s2/blob/ebf4b28e3a047a39888aabbb31f60100fddf2729/proto/public/s2/v1alpha/s2.proto#L391-L392

See https://github.com/s2-streamstore/s2-sdk-rust/issues/152